src/convertToObject.js

```diff
@@ -1,18 +1,33 @@
 'use strict';
-
+'eslint-disable no-console';
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in [stylesString.js]
+ * (./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStyles = sourceString
+    .split(';')
+    .map(style => style.trim())
+    .filter(style => style !== '');
+
+  const objectOfStyles = arrayOfStyles.reduce((obj, style) =>
+    (
+      {
+        ...obj, [style.split(':')[0].trim()]: style.split(':')[1].trim(),
+      }
+    ), {});
```

Comment:
you can create variables for
`style.split(':')[0].trim()` and `style.split(':')[1].trim()`
to make it more visually beautiful

and also move to a new line after `...obj,`